package.json

```diff
@@ -5,67 +5,76 @@
     "type": "module",
     "version": "1.0.0",
     "scripts": {
-        "build": "remix build",
-        "deploy": "wrangler deploy --var VERSION:$(git rev-parse HEAD)",
-        "dev": "remix dev --manual -c \"npm start\"",
+        "dev": "remix vite:dev",
+        "build": "remix vite:build",
+        "prepreview": "remix vite:build",
+        "preview": "wrangler pages dev ./build/client",
```

Comment:
When should I use `npm run preview` and when should I use `npm run dev`? Are the `remix` commands deprecated now?

Reply:
`wrangler pages dev ./build/client` command simulate the CF environment locally like functions
Where as `remix vite:build` just works with remix, usually this command alone will have 404 for `/collect` endpoint of CF functions.

If we manage to migrate `/collect` endpoint to remix, then preview wouldn't be needed anymore. I believe this is doable with minor changes, I will try to get this done in couple of days.